lib/baseTest.py

```diff
@@ -76,46 +76,46 @@ def get_browser_done(self):
         recording_enabled = CommonUtil.is_video_recording(self.firefox_config)
 
         if recording_enabled:
+            viewport = dict()
             viewport_ref_fp = os.path.join(self.env.img_sample_dp, self.env.img_output_sample_1_fn)
             is_expected_viewport = False
-            for i in range(10):
+            time.sleep(3)
```

Comment:
Sleeping 3 sec without do something is wasting time.
Increasing the loop time from `20` to `25` might be the alternative solution.